apis/bases/network.openstack.org_dnsmasqs.yaml

```diff
@@ -185,6 +185,11 @@ spec:
                   - type
                   type: object
                 type: array
+              dnsAddresses:
+                description: DNSServer Addresses
+                items:
+                  type: string
+                type: array
```

Comment:
thinking about if this needs to be an array. the IP address for dns is a single metallb vip. but it won't hurt to use an array.

Reply:
right we use also a list when requesting IPs via the CRD.